packages/ai-mcp/src/node/mcp-server.ts

```diff
@@ -84,23 +95,41 @@ export class MCPServer {
         if (this.isRunnning() && this.status === MCPServerStatus.Starting) {
             return;
         }
+
         this.setStatus(MCPServerStatus.Starting);
-        console.log(`Starting server "${this.name}" with command: ${this.command} and args: ${this.args?.join(' ')} and env: ${JSON.stringify(this.env)}`);
-        // Filter process.env to exclude undefined values
-        const sanitizedEnv: Record<string, string> = Object.fromEntries(
-            Object.entries(process.env).filter((entry): entry is [string, string] => entry[1] !== undefined)
-        );
-
-        const mergedEnv: Record<string, string> = {
-            ...sanitizedEnv,
-            ...(this.env || {})
-        };
-        const transport = new StdioClientTransport({
-            command: this.command,
-            args: this.args,
-            env: mergedEnv,
-        });
-        transport.onerror = error => {
+        console.log(`Starting server "${this.name}" with transport: ${this.transportType}`);
```

Comment:
We log two messages before we start a server now. I think we can combine it to one covering both cases. What I mean: I would merge the content of the two messages below into the message here so that we get only one log message including all information (the URL for SSE, the command, args and env for stdio)